bluepyemodel/evaluation/fitness_calculator_configuration.py

```diff
@@ -105,24 +105,55 @@ def __init__(
                 .. code-block::
 
                     [
-                        {"efel_feature_name": str, "protocol_name": str, "recording_name": str,
-                        "mean": float, "std": float, "efel_settings": dict}
+                        {
+                            "efel_feature_name": "Spikecount",
+                            "protocol_name": "IDrest_130",
+                            "recording_name": "soma.v",
```

Comment:
Did you forget to include the protocol_name here? In the previous comment, you mentioned that "here, recording_name should have the format ``f"{protocol_name}.{location}.v"``,
                e.g. ``"IV_0.soma.v"``"

Reply:
I mixed up the recording name which had the protocol name, and the one that did not in the previous PR ^^'